openapi/doc.go

```diff
@@ -0,0 +1,3 @@
+package openapi
+
+// adding a zero dep package so that we can vendor and pull the openapi json out of here.
```

Comment:
> I don't get what this is achieving, can you expand?

We need this doc package to be vendored into client-go.

Reply:
You can make it require the `cmd/models-schema` package and it will import it, I've done that before

Eg I think this in a tools.go will ensure it's vendored in the client-go vendor dir, even without this doc.go file. Not a blocker though

```
import (
  _ "github.com/openshift/api/openapi/cmd/models-schema"
)
```